CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Cromwell Change Log
 
+## 80 Release Notes
+
+As a means to stay on top of endpoints within our repo, 3 new Workflow Execution Service (WES) endpoints are now documented in the Cromwell Swagger (others to follow as part of later work):
+
+| HTTP verb | Endpoint path | Description   |
+| --------- | ------------- |---------------|
+| GET | /service-info | Server info |
+| POST | /runs/{run_id}/cancel | Abort workflow |
+| GET | /runs/{run_id}/status | Workflow status |
+
 ## 79 Release Notes
```

Comment:
Oops, I was ambiguous in Slack about version number.

The current shipping release is 78: https://github.com/broadinstitute/cromwell/releases

These changes should go under the 79 heading. When you see a version like `79-841bf43-SNAP`, the `-SNAP` means that it is a pre-release work in progress that will be called `79-xxxxxxx` (no `-SNAP`) when it ships.

I also think we should probably include the full path `/api/ga4gh/wes/v1/` in the endpoint.

```suggestion
## 79 Release Notes

### WES endpoints preview

As a means to stay on top of endpoints within our repo, 3 new Workflow Execution Service (WES) endpoints are now documented in the Cromwell Swagger (others to follow as part of later work):

| HTTP verb | Endpoint path | Description   |
| --------- | ------------- |---------------|
| GET | /api/ga4gh/wes/v1/service-info | Server info |
| POST | /api/ga4gh/wes/v1/runs/{run_id}/cancel | Abort workflow |
| GET | /api/ga4gh/wes/v1/runs/{run_id}/status | Workflow status |
```